### DISCUSSION TITLE
IOS端服务器不停的来回切换

### DISCUSSION THREAD

**Discussion by airou87** [getlantern/lantern-forum-cn#1110] [General]
IOS端现在虽然能用了，但服务器很不稳定，总是在韩国日本新加坡洛杉矶之间来回切换，导致我脸书账号都被封锁了。什么时候出手动选服务器功能

**Comment by MaxMKMax** [comment]
@airou87 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@airou87 其实技术团队又给你们弄稳定服务器了. 不知道为什么这个问题一直在. 我再帮您反馈吧.

**Reply by airou87** [reply]
app里提交不了问题报告，提示错误。尽快出手动选服务器功能吧

**Comment by MaxMKMax** [comment]
@airou87 您好. 谢谢您的反馈. 我会帮您反馈上去的. 请您用以下链接的方法帮我们手动提交log可以吗? 麻烦您了. 非常谢谢您!
https://github.com/getlantern/guide/wiki/Troubleshooting#application-issue-report

**Comment by MaxMKMax** [comment]
@airou87 因为我们技术团队需要查看log才能够找到问题所在. 所以需要您的帮忙. 谢谢啦!

**Comment by MaxMKMax** [comment]
@airou87 您好. 您记得帮我们提交 LOG 报告哦. 麻烦您了. 谢谢啦!

**Comment by MaxMKMax** [comment]
@airou87 您好. 您记得帮我们提交 LOG 报告哦. 方便我明天和美国团队开会讨论您遇到的问题. 谢谢啦!

**Reply by airou87** [reply]
![image](https://github.com/user-attachments/assets/f8c85a76-6217-440f-ac63-465fcbadea64)

**Comment by MaxMKMax** [comment]
@airou87 请问您使用哪一型的手机号是电脑遇到无法提交应用内报告的问题呢? 

IOS? Android? Window? MacOS? 其他? 
和您确认一下就只有iOS有这个问题吗?

**Reply by airou87** [reply]
手机型号iPhone 16 pro 无法提交报告

**Comment by MaxMKMax** [comment]
@airou87 
您好. 技术团队说您好像不是最新版本. 请您用这个链接更新到最新的版本.
https://github.com/getlantern/guide/wiki/Downloads-&-Updates#updating

**Comment by MaxMKMax** [comment]
@airou87 您好. 请问您还有遇到以上的两个问题吗? 服务器切换和无法提供应用内报告.

您好. 技术团队说您好像不是最新版本. 请您用这个链接更新到最新的版本.
https://github.com/getlantern/guide/wiki/Downloads-&-Updates#updating

**Comment by MaxMKMax** [comment]
@airou87 您好. 请问您还有遇到以上的两个问题吗? 服务器切换和无法提供应用内报告的问题.
